job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -335,6 +337,55 @@ definitions:
         example:
           "tag:4": "4"
           owner: "bob"
+  expirationPolicy:
+    type: object
+    properties:
+      "Active":
+        $ref: "#/definitions/policy"
+      "Completed":
+        $ref: "#/definitions/policy"
+      "Failed":
+        $ref: "#/definitions/policy"
+      "Cancelled":
+        $ref: "#/definitions/policy"
+      "Waiting":
+        $ref: "#/definitions/policy"
+      "Paused":
+        $ref: "#/definitions/policy"
+      "Expired":
+        $ref: "#/definitions/deletePolicy"
+      "Default":
+        $ref: "#/definitions/policy"
+    description: The expiration policy to be applied on the job
+  deletePolicy:
+    type: object
+    properties:
+      expiryTime:
+        type: string
+        pattern: '^((lastUpdateDate)|(createDate))(\+P)([1-9]\d?|[12]\d{2}|3[0-5]\d|36[0-5])D$'
+        default: "createDate+10D"
+      expirationOperation:
+        type: string
+        enum:
+          - delete
+    description: The expiration details to be applied on the job
+  policy:
+    type: object
+    properties:
+      expiryTime:
+        type: string
+        pattern: '^((lastUpdateDate)|(createDate))(\+P)([1-9]\d?|[12]\d{2}|3[0-5]\d|36[0-5])D$'
+        default: "createDate+10D"
+      expirationOperation:
+        $ref: "#/definitions/expirationOperation"
+    description: The expiration details to be applied on the job
+  expirationOperation:
```

Comment:
You're mixing up definition names and property names.  Definition names are internal and flat.  Property names are public and have a context.  Just because you changed this to `expirationOperation` it doesn't follow that the property name should have changed.

Reply:
replaced with operation